packages/manager/apps/cloud-connect/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect-app",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cloud-connect-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-cloud-connect": "^0.0.0 || ^1.0.0",
+    "@ovh-ux/manager-config": "^0.4.0",
+    "@ovh-ux/manager-core": "^9.0.0 || ^10.0.0",
```

```suggestion
    "@ovh-ux/manager-core": "^10.0.0 || ^11.0.0",
```
I think a breaking change has been made on core